app/Http/ViewComposers/Frontend/Tool/SimpleScanComposer.php

```diff
@@ -29,17 +30,15 @@ public function create(View $view)
             $subStep = $step->subSteps()->orderByDesc('order')->first();
         }
 
-        $total = Step::quickScan()
+        $total = $scan
+            ->steps()
             ->leftJoin('sub_steps', 'steps.id', '=', 'sub_steps.step_id')
             ->count();
 
         // Get all the IDs of previous steps. Currently there are a total of 4 steps, so
         // we can have a maximum of 3 step IDs
```

Comment:
This comment is no longer relevant; might be confusing in the future as there's more than 4 steps now